templates/security.html

```diff
@@ -0,0 +1,24 @@
+<article class="center security">
+  <h1>Security</h1>
+  <div class="wrap" style="padding-bottom:30px;">
+    <p><b>lib</b>p2p is still under active development and there may be
+    problems with our protocols or errors in our implementations. We take
+    security vulnerabilities very seriously and follow responsible disclosure
+    best practices. If you discover a security issue, please bring it to our
+    attention right away!</p>
+  </div>
+  <h2>Reporting a Security Vulnerability</h2>
+  <div class="wrap" style="padding-bottom:30px;">
+      <p>If you find a vulnerability that may affect live deployments—such as a
+      remote execution exploit—please <a href="https://github.com/libp2p/go-libp2p/security/advisories/new">report it privately</a>.
```

Comment:
Should we rephrase to link to each implementation's advisories? Or maybe just say to reach out to the email and we can triage.

Reply:
I think the email is easier for a central security team to manage.

Reply:
OH! I see, yeah, the link is for go-libp2p. Hrm...I think maybe we just say, "report it privately by sending an email to security@libp2p.io. DO NOT file a public issue"